docs/skopeo-sync.1.md

```diff
@@ -235,6 +239,7 @@ This will copy the following images:
 - Repository `registry.example.com/redis`: images tagged "1.0" and "2.0" along with image with digest "sha256:0000000000000000000000000000000011111111111111111111111111111111".
 - Repository `registry.example.com/nginx`: images tagged "1.13.1-alpine-perl" and "1.13.2-alpine-perl".
 - Repository `quay.io/coreos/etcd`: images tagged "latest".
+- Repository `registry.example.com/alpine`: all images with tags satisfying either the "3.12 - 3.13" condition ("3.12.0", "3.12.1"...) or the ">= 3.17" ("3.17.5", "3.19.0", "4.0.0"...)
```

Comment:
This should, somehow, point at the full relevant documentation of the specific constraint format. Maybe just a link to the website / repo.

Reply:
Agreed, added a link to upstream documentation for version comparison and actual range syntax.